atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleNamespaceDeleterIntegrationTest.java

```diff
@@ -57,15 +57,20 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import org.immutables.value.Value;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.ClassRule;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-public final class OracleNamespaceDeleterIntegrationTest extends TransactionTestSetup {
+@ExtendWith(DbKvsOracleExtension.class)
+@Order(1)
```

Comment:
If `OracleNamespaceDeleterIntegrationTest` does not run first, then some of its methods fail. Although this solution works, I couldn't find the root cause. Considering that I cannot run Oracle integration tests locally, further debugging may take much more time (as I need to wait for CircleCI each time I try something).

Note that it is probably something to do with we are using the same docker extension for everything.